valkyrie.drush.inc

```diff
@@ -121,7 +121,11 @@ function valkyrie_drush_command() {
     'aliases' => array('vup'),
     'bootstrap' => DRUSH_BOOTSTRAP_DRUSH, // No bootstrap at all.
   );
-
+  $items['valkyrie-upgrade'] = array(
+    'description' => 'Upgrade global Valkyrie extension and all of the submodules.',
+    'aliases' => array('vug'),
```

Comment:
by default, this should probably be equivalent to 'drush dl valkyrie'. That is, download and install the release tarball. Maybe add a --git option, to optionally switch to, or update a git version.